### ISSUE TITLE
[Duke Nukem 64] Screen fades/flashes from item pickup, damage, level fadeins, etc, are not displaying.

### ISSUE THREAD

**Issue by Terraformer9x** [nukeykt/NBlood#756]
The colored screen flashes and fades for picking up items, taking damage, and other events are not displaying properly and instead only color the weapon. This happens on both 32 and 64 bit. This bug is not present in the release build (r13623) on the lerppu site.

**Example 1: Item Pickup**
Taken in r13623, screen flashes blue and weapon tints blue after picking up an item.
![rd1](https://github.com/nukeykt/NBlood/assets/83795041/14942628-a428-428c-8b03-e7e13f095903)

Taken in r13800, picking up an item does not yield a blue screen flash, but the weapon still tints blue.
![rd2](https://github.com/nukeykt/NBlood/assets/83795041/86361870-da1e-403c-9f98-bbf67d12607f)

**Example 2: Damage**
Taken in r13623, screen flashes red and weapon tints red after taking damage.
![rd3](https://github.com/nukeykt/NBlood/assets/83795041/943ddf02-0ae7-4fc3-81bb-20210f9b410e)

Taken in r13800, taking damage does not yield a red screen flash, but the weapon still tints red.
![rd4](https://github.com/nukeykt/NBlood/assets/83795041/dfcad01f-a900-4172-87be-1a5e3fd0ed79)

**Comment by Hendricks266** [comment]
Cross-posting from Discord, this was introduced in merge commit 2a8bbae9fdbb66f7c33521b8e75e19347ee20b7b.

**Comment by DarkMasterMan0** [comment]
I can confirm this is still bugged as of r14192, there's still no blue screen tint upon picking up items nor a red screen tint upon taking damage, the tints only appear on the weapon.

**Comment by Flpstrike** [comment]
Yeah... that has been a annoying issue that makes it hard to receive feedback when Duke takes damage.
